docs/source/examples/workspaces/openshift/PinFile

```diff
@@ -8,7 +9,7 @@ openshift-new:
         resource_definitions:
           - name: testsecret
             role: openshift_inline
-            namespace: continuous-infra
+            namespace: {{ namespace | default('continuous-infra') }}
```

Comment:
The jinja vars have to be in " " else ansible won't recognize(flaky issue w/ ansible).